src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -50534,10 +50579,11 @@ void M1()
 class C {}
 ";
             var comp = CreateCompilation(new[] { source });
-            // https://github.com/dotnet/roslyn/issues/30220: Missing warnings for
-            // B<T1?> b1;
```

Comment:
No warning for this one?

Reply:
> No warning for this one?

Not on this one because containing type of ```B<T1?>``` is a definition, therefore the ```T1``` constraint remains oblivious due to the "#nullable disable" on the declaration of ```B```.

---
In reply to: [227144844](https://github.com/dotnet/roslyn/pull/30616#discussion_r227144844) [](ancestors = 227144844)